libs/oidc/common/src/lib/services/user/user.service.ts

```diff
@@ -43,28 +45,8 @@ export class UserService {
     private readonly httpService: HttpService
   ) {}
 
-  public async insertDefaultAdmin(email: string, password: string) {
-    const _user: Partial<User> = {
-      email: email,
-      password: password,
-      signup_ip_address: '1.1.1.1',
-      last_used_ip_address: '1.1.1.1',
-      updatedAt: new Date(),
-      added: new Date()
-    };
-
-    const user = await this.userRepo.create(_user);
-    const entUser = await this.insertUser(user, 'admin');
-
-    // Set Admin role for admin user
-    const adminRole = await this.roleRepo.findOne({
-      where: {
-        level: '99'
-      }
-    });
-
-    console.log('Admin userGuid:', entUser.guid);
-    await this.insertUserRole(entUser, adminRole, 'oidc-idp-admin');
+  public async insertDefaultAdmin() {
+    throw new NotImplementedException();
```

Comment:
Can this be? It' can't possibly....

Old implementation removed entirely instead of commenting out huge chunks of code with a "TODO:"?!?!?!

The gods have answered my prayers! Hallelujah!!!!

Reply:
👍